cpp/src/bindings.cpp

```diff
@@ -143,6 +143,29 @@ std::shared_ptr<load_flow_parameters> initLoadFlowParameters() {
     });
 }
 
+std::shared_ptr<security_analysis_parameters> initSecurityAnalysisParameters() {
+    security_analysis_parameters* parameters = new security_analysis_parameters();
+    std::shared_ptr<security_analysis_parameters> config_params = pypowsybl::createSecurityAnalysisParameters();
+    parameters->flow_proportional_threshold = config_params->flow_proportional_threshold;
+    parameters->low_voltage_proportional_threshold = config_params->low_voltage_proportional_threshold;
+    parameters->low_voltage_absolute_threshold = config_params->low_voltage_absolute_threshold;
+    parameters->high_voltage_proportional_threshold = config_params->high_voltage_proportional_threshold;
+    parameters->high_voltage_absolute_threshold = config_params->high_voltage_absolute_threshold;
+    parameters->provider_parameters_keys_count = config_params->provider_parameters_keys_count;
+    parameters->provider_parameters_values_count = config_params->provider_parameters_values_count;
+    return std::shared_ptr<security_analysis_parameters>(parameters, [](security_analysis_parameters* ptr){
+    });
+}
+
+std::shared_ptr<sensitivity_analysis_parameters> initSensitivityAnalysisParameters() {
+    sensitivity_analysis_parameters* parameters = new sensitivity_analysis_parameters();
+    std::shared_ptr<sensitivity_analysis_parameters> config_params = pypowsybl::createSensitivityAnalysisParameters();
+    parameters->provider_parameters_keys_count = config_params->provider_parameters_keys_count;
+    parameters->provider_parameters_values_count = config_params->provider_parameters_values_count;
+    return std::shared_ptr<sensitivity_analysis_parameters>(parameters, [](sensitivity_analysis_parameters* ptr){
+    });
```

Comment:
Like for security analysis : we need to free the parameters memory here, including the loadflow parameters memory.